data/data/bootstrap/systemd/common/units/kubelet.service.template

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=Kubernetes Kubelet
-Wants=rpc-statd.service crio.service release-image.service
-After=crio.service release-image.service
+Wants=rpc-statd.service crio.service release-image.service{{if .IsOKD}} release-image-pivot.service{{end}}
```

Comment:
```suggestion
Wants=rpc-statd.service crio.service release-image.service
{{if .IsOKD}}Wants=release-image-pivot.service{{end}}
```

same functionality but cleaner to read / revert

Reply:
Good idea, ty!